redis_collections/__init__.py

```diff
@@ -1,5 +1,5 @@
 __title__ = 'redis-collections'
-__version__ = '0.11.0'
+__version__ = '0.12.0'
 __author__ = 'Honza Javorek'
```

Comment:
Is there a way to include you too? 🤔

Reply:
Looks like the core metadata spec allows for a [Maintainer](https://packaging.python.org/en/latest/specifications/core-metadata/#maintainer), which fits.

Reply:
Go for it :)